src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -31,29 +32,17 @@ public interface IInvocationExpression : IExpression
         /// <summary>
         /// 'This' or 'Me' argument to be supplied to the method.
         /// </summary>
-        IExpression Instance { get; }
+        IOperation Instance { get; }
```

Comment:
Mentioned this with @JohnHamby offline, this would return null for calls to static method. Maybe worth mentioning in the comment.